ksql-common/src/main/java/io/confluent/ksql/util/KsqlHost.java

```diff
@@ -42,8 +43,8 @@ public boolean equals(final Object o) {
       return false;
     }
 
-    final KsqlHost hostInfo = (KsqlHost) o;
-    return port == hostInfo.port && host.equals(hostInfo.host);
+    final KsqlHost host = (KsqlHost) o;
+    return this.host.equals(host.host) && port == host.port;
```

Comment:
can we rename `host` to `otherHost` or `thatHost` for better readability?

Reply:
Another suggestion.. Can we name this class `KsqlHostPort` or `KsqlHostInfo`, since it already has a member `host`